src/lib/util/statistics.ts

```diff
@@ -0,0 +1,408 @@
+import {
+  QuestionnaireByID,
+  ResponseResult,
+  QuestionDetails,
+  ResponseBody
+} from '/@/lib/apis'
+
+export const adjustQuestions = (
+  questionnaire: QuestionnaireByID,
+  questions: QuestionDetails[],
+  results: ResponseResult[]
+): ResultsPerQuestion => {
+  const resQuestions: BaseTypeQuestion[] = questions.map(q => {
+    const resultsPerQuestionWithUser = results
+      .map(r =>
+        r.body.map(body => ({
+          ...body,
+          modified_at: r.modified_at,
+          submitted_at: r.submitted_at,
+          traqID: r.traqID
+        }))
+      )
+      .map(bodies => bodies.find(v => v.questionID === q.questionID))
+      .filter((v): v is Exclude<typeof v, undefined> => !!v)
+    return {
+      type: q.question_type,
+      question: q,
+      results: resultsPerQuestionWithUser
+    }
+  })
+  const validTypeQuestion: AllTypeQuestionUnion[] = []
+  resQuestions.forEach(function (array) {
+    if (isValidTypeQuestion(array)) {
+      validTypeQuestion.push(array)
+    }
+  })
```

Comment:
```suggestion
  resQuestions.forEach(v => {
    if (isValidTypeQuestion(v)) {
      validTypeQuestion.push(v)
    }
  })
```
ここの関数に渡される引数は`array`(配列)じゃなくて各要素なので変数名は変えてほしいです
長いスコープで使われるわけじゃないので僕はよく`value`の頭文字の`v`を使います

あと他はarrow functionで書いてるので統一してほしいです

Reply:
> あと他はarrow functionで書いてるので統一してほしいです

がんばります


> わかんなかったら具体的に言うんでいってください

具体的にお願いします:pray: